### PR TITLE
Deploy travis

### LINKED ISSUE #369: auto deploy with travis
auto deploy from **travis** to **mainnet** and **rinkeby** upon merging to master  (**staging** and **production**)
using the following links: 
 
https://thegraph.com/explorer/subgraph/daostack/master 
https://thegraph.com/explorer/subgraph/daostack/master_rinkeby
https://staging.thegraph.com/explorer/subgraph/daostack/master
https://staging.thegraph.com/explorer/subgraph/daostack/master_rinkeby

### PR DESCRIPTION
**Pull request by ben-kaufman**
Fix #369

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,3 +36,7 @@ script:
 
   - npm run lint
   - npm run jest
+
+after_success:
+- if [ "$TRAVIS_BRANCH" = "master" ]; then
```
Comment (orenyodfat) [inline .travis.yml]: is that been tested vs other branch first ?

File: .travis.yml [hunk 2]
```diff
@@ -36,3 +36,7 @@ script:
 
   - npm run lint
   - npm run jest
+
+after_success:
+- if [ "$TRAVIS_BRANCH" = "deploy-travis" ]; then
```
Comment (orenyodfat) [inline .travis.yml]: does TRAVIS_BRANCH is always the 'current' branch that travis works on ?

Comment (orenyodfat) [inline .travis.yml]: if [ $TRAVIS_BRANCH = "deploy-travis" ]; then

File: deploy-travis.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+if [ ! -z "$access_token" ]; then
```
Comment (orenyodfat) [inline deploy-travis.sh]: does access_token is available only for master ?

Comment (ben-kaufman) [reply deploy-travis.sh]: We can limit it to master, but anyway we have an if condition in the travis script to limit it to master (or other specific branch)

**Pull request merged**